frontend/src/pages/question/questionBuilder/useNewQuestionPanel.js

```diff
@@ -125,6 +125,10 @@ export default function useNewQuestionPanel() {
    * @param {string} id unique id of node or edge. i.e. n0, n1, e0...
    */
   function openPanel(type, id) {
+    if (type === 'node' && query_graph.nodes[id] && query_graph.nodes[id].deleted) {
```

Comment:
I'm not sure why but I am still able to open the NodePanel for a deleted node. I created a query graph with Disease -> Gene, deleted gene, and was still able to open gene.

Reply:
It looks like query_graph is empty for some reason when the panel is opened.

Reply:
Weird, ok. I'll look into this some more.